src/client/app/utils/input.ts

```diff
@@ -61,7 +61,10 @@ export function potentialGraphicUnits(units: UnitDataById) {
 		}
 	});
 	// Put in alphabetical order.
-	possibleGraphicUnits = new Set(sortBy(Array.from(possibleGraphicUnits), unit => unit.identifier.toLowerCase(), 'asc'));
+	// TODO: change second argument of locale from undefined to current language if needed, but alphabetical ordering works
```

Comment:
See elsewhere on thoughts about possibly resolving this.

Reply:
Finished implementation of changes.